addons/captives/functions/fnc_canUnloadCaptive.sqf

```diff
@@ -19,4 +19,6 @@
 params ["_player", "_unit"];
 
 // Don't show "Unload Captive" if unit is unconscious (already has "Unload Patient")
-(vehicle _unit != _unit) && {vehicle _player == _player} && {_unit getVariable [QGVAR(isHandcuffed), false]} && {!(_unit getVariable ["ACE_isUnconscious", false])}
+!isNull objectParent _unit &&
+{_unit getVariable [QGVAR(isHandcuffed), false]} &&
+{lifeState _unit in ["HEALTHY", "INJURED"]}
```

Comment:
I realise this was already the case pre-PR, but for this last condition, surely we only want to respect it if medical is loaded (otherwise there'd be no unload action)?

Reply:
Vanilla has a pull out bodies/uncon action

Reply:
Ah perfect, shows when I last booted up arma :laughing: